script.js

```diff
@@ -1,4 +1,9 @@
 let projectData = [
+ {
+  projectname: 'And Or Xor Converter',
+  projectUrl: 'public/AND,OR,XOR Converter.html',
```

Comment:
- Don't insert comma and space in filename.
- Add gif to the project.

Fix it please